### PR TITLE
Updating Standard licensing Files

### PR DESCRIPTION
**Pull request by BhargaviPalineni79**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Contributing:
+
+If you would like to contribute code to this project you can do so through GitHub by forking the repository andsending a pull request. Before RDK accepts your code into the project you must sign the RDK Contributor License Agreement (CLA).
```
Comment (mhughesacn) [inline CONTRIBUTING.md]: typo: "andsending" missing a space.

File: COPYING [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/bhargavi/979/1050/Lightning-Automotive/LICENSE
```
Comment (mhughesacn) [inline COPYING]: This link should be to LICENSE, not this specific.

**Review by mhughesacn — COMMENTED** [review]
Hi Bhargavi: A few small things please.  The typo in CONTRIBUTING may be elsewhere but may as well be right here. Thanks.

**Review by mhughesacn — APPROVED** [review]
Thanks Bhargavi

**Review by wouterlucas — APPROVED** [review]
LGTM

**Pull request merged**